### PR TITLE
load super token by symbol

### PR DESCRIPTION
**Pull request by 0xdavinchee**

### CODE REVIEW

File: packages/sdk-core/src/Framework.ts [hunk 1]
```diff
@@ -250,10 +251,31 @@ export default class Framework {
...
+    loadSuperToken = async (
+        tokenAddressOrSymbol: string
+    ): Promise<SuperToken> => {
+        let address;
+        const isValidAddress = ethers.utils.isAddress(tokenAddressOrSymbol);
+
+        if (isValidAddress) {
+            address = tokenAddressOrSymbol;
+        } else {
+            const superTokenKey = `supertokens.${this.settings.protocolReleaseVersion}.${tokenAddressOrSymbol}`;
```
Comment (hellwolf) [inline packages/sdk-core/src/Framework.ts:266]: error handling

Comment (0xdavinchee) [reply packages/sdk-core/src/Framework.ts]: added a try/catch

File: packages/sdk-core/src/Framework.ts [hunk 2]
```diff
@@ -250,10 +251,47 @@ export default class Framework {
 
     /**
      * @dev Load a `SuperToken` class from the `Framework`.
-     * @param address the `SuperToken` address
+     * @param tokenAddressOrSymbol the `SuperToken` address or symbol
```
Comment (kasparkallas) [inline packages/sdk-core/src/Framework.ts:254]: Should mention that the symbol has to be listed for this to work?

**Review by kasparkallas — APPROVED** [review]

File: packages/sdk-core/src/Framework.ts (hunk 2 above)
Comment (0xdavinchee) [reply packages/sdk-core/src/Framework.ts:254]: true

**Pull request merged**